app/logical/tag_query_new.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+class TagQueryNew < TagQuery
+  class UnclosedGroupError < StandardError; end
+  class MaxGroupDepthExceededError < StandardError; end
+  class MaxTokensAchievedInGroupError < StandardError; end
+
+  attr_reader :q, :resolve_aliases
```

Comment:
you don't need to redefine the readers since you're extending the base

Reply:
Fixed 0c2b9d9156a63ecd5deeafe1d76162a270144367